Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -280,6 +286,13 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   // workflow in the upper left corner of the GUI.
   WorkflowSpec specs(1);
 
+  // Update the (declared) parameters if changed from the command line
+  // Note: In the future this should be done only on a dedicated processor managing
+  // the parameters and then propagated automatically to all devices
```

Comment:
Do you mean to keep these line here only to write the ``ini`` file and to add to all devices which may need configuration:
in ``DataProcessorSpec getXXXSpec()``:
````
Options{
...
{ "configKeyValues", VariantType::String, "", { "KeyValue params" } }
}
````
and in the ``init()``:
``o2::conf::ConfigurableParam::updateFromString( ic.options().get<std::string>("configKeyValues")) ;`` ?

Reply:
Something along those lines...